slack/notify-result/README.md

```diff
@@ -0,0 +1,29 @@
+# <!--name-->slack/notify-result<!--/name-->
+
```

Comment:
```suggestion
<!--description-->
```

needs an gh action-readme update

Reply:
what do you mean? I used the pre-commit locally

Reply:
or pre-commit, yeah 😅

Reply:
Sorry, I wasn't clear enough.

The description is missing in the `README.md`.

I added the suggestion to add `<!--description->` tag.

But if you apply the suggestion, it probably still needs to be updated by pre-commit or gh action-readme.